pocs/tests/test_filterwheel.py

```diff
@@ -0,0 +1,148 @@
+import pytest
+import math
+from timeit import timeit
+import time
+
+from astropy import units as u
+
+from pocs.filterwheel.simulator import FilterWheel as SimFilterWheel
+from pocs.camera.simulator import Camera as SimCamera
+
+
+@pytest.fixture(scope='module')
+def filterwheel():
+    sim_filterwheel = SimFilterWheel(filter_names=['one', 'deux', 'drei', 'quattro'],
+                                     move_time=0.1 * u.second,
+                                     timeout=0.5 * u.second)
+    return sim_filterwheel
+
+# intialisation
+
+
+def test_init(filterwheel):
+    assert isinstance(filterwheel, SimFilterWheel)
+    assert filterwheel.is_connected
+
+
+def test_camera_init():
+    sim_camera = SimCamera(filterwheel={'model': 'simulator',
+                                        'filter_names': ['one', 'deux', 'drei', 'quattro']})
+    assert isinstance(sim_camera.filterwheel, SimFilterWheel)
+    assert sim_camera.filterwheel.is_connected
```

Comment:
Maybe we want to check these values in the case when there is no filterwheel as well?

Reply:
Well, a camera with no filterwheel will have a `filterwheel` attribute (thanks to the `_create_subcomponent()` call) but the value will be `None`. Adding a test for that.